### PR TITLE
Sort icons and remove unnecessary full path from Edit icons

### PR DESCRIPTION
**Pull request by padvincenzo**
Hi @Amstutz 
I've made little changes to your recent commit about the icons, hope you will appreciate them.

Bye,
Vincenzo

### CODE REVIEW

File: Services/Style/System/classes/Icons/class.ilSystemStyleIconFolder.php [hunk 1]
```diff
@@ -76,6 +76,16 @@ protected function sortIcons()
         });
     }
 
+    /**
+     * Sorts the Icons by path
+     */
+    public function sortIconsByPath()
+    {
+        usort($this->icons, function (ilSystemStyleIcon $a, ilSystemStyleIcon $b) {
```
Comment (mjansenDatabay) [inline Services/Style/System/classes/Icons/class.ilSystemStyleIconFolder.php]:
```suggestion
        usort($this->icons, static function (ilSystemStyleIcon $a, ilSystemStyleIcon $b) : int {
```

File: Services/Style/System/classes/Icons/class.ilSystemStyleIconFolder.php [hunk 2]
```diff
@@ -76,6 +76,16 @@ protected function sortIcons()
         });
     }
 
+    /**
+     * Sorts the Icons by path
+     */
+    public function sortIconsByPath()
```
Comment (mjansenDatabay) [inline Services/Style/System/classes/Icons/class.ilSystemStyleIconFolder.php]:
```suggestion
    public function sortIconsByPath() : void
```

File: Services/Style/System/classes/Icons/class.ilSystemStyleIconFolder.php [hunk 3]
```diff
@@ -76,6 +76,16 @@ protected function sortIcons()
         });
     }
 
+    /**
+     * Sorts the Icons by path
+     */
```
Comment (mjansenDatabay) [inline Services/Style/System/classes/Icons/class.ilSystemStyleIconFolder.php]:
```suggestion
```

**Review by Amstutz — APPROVED** [review]
Hi @padvincenzo 

Thx a lot for your addition/improvement. Highly appreciated. Can you add the suggested changes by @mjansenDatabay (thx). 

Ill merge your improvements afterwards. Kudos and thx.

**Comment by padvincenzo** [comment]
Hi @Amstutz  
I've just added those changes. Thanks @mjansenDatabay for the review 😃

**Comment by Amstutz** [comment]
Thx @padvincenzo , nice work. The dropdown looks a lot cleaner/more readable now, which improves the usability of this view by quite a bit. Merged and pushed to 7 and trunk.

**Pull request merged**